ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -16,7 +16,10 @@ def det(input, name=None):
 
 
 def eigh(tensor, name=None):
```

Comment:
This was already implemented :)

Reply:
Hi James! Thank you for your response. The change I made to the already implemented eigh function was that the tensorflow function returns two variables, i.e., the eigen values and the eigen vectors, while the ivy.eigh function returns one vairable that comprises of a tuple containing eigen values and eigen vectors of the input. So, to better mimic the tensorflow function, I separated the tuple into two variables.

Reply:
Oh I see! My bad, that makes sense. Good find!

Reply:
Did you implement the previous eigh() for the tensorflow frontend?

Reply:
Thank you for your feedback. I did not implement the previous eigh function, however, it was not present when I opened the issue.

Reply:
It seems to be the test `test_tensoflow_eigh` is still failing, this would need to be fixed before I can merge this PR :)